Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -42,10 +42,19 @@ protected class MemoryCacheItem
             public BitmapImage Image { get; set; }
         }
 
+        private class ConcurrentTask
+        {
+            public Task<BitmapImage> Task { get; set; }
+
+            public bool IsAssureOnly { get; set; }
```

Comment:
What about IsPreCache, since the public method is called PreCacheAsync? Anyway I think we should align the name with public method, but I don't know which is more appropriate "PreCacheAsync" or "EnsureCachedAsync".